relnotes/formatter_comma_support.feature.md

```diff
@@ -0,0 +1,6 @@
+* `ocean.text.convert.Formatter`
+
+  Use `{_}` format specifier to print a comma `,` after every 3rd digit when
+  printing a decimal integers or a floating point number.  If combined with a
+  hex-number specifier `{:x_}` or with a binary number specifier  `{:b_}`, then
```

Comment:
I think it might make sense to separate  every 2 digits for hex and 8 for binary (and 4 for octal), and maybe not using a comma for them, but a space or a dash maybe.